packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -5539,15 +5539,7 @@ describe('plugin-meetings', () => {
             }
           };
           meeting.parseMeetingInfo(mockToggleOnData);
-          assert.calledWith(updateHostSIEnabledSpy, true);
-
-          const mockToggleOffData = {
-            body: {
-              meetingSiteSetting: {}
-            }
-          };
-          meeting.parseMeetingInfo(mockToggleOffData);
-          assert.calledWith(updateHostSIEnabledSpy, false);
+          assert.calledOnce(parseInterpretationInfo);
```

Comment:
issue: the arguments for this call should be checked. You have access to both of them in the test i.e. the meeting and the body of the mock toggle on data

Reply:
Updated